### PR TITLE
#37575 case search field configuration

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

**Review by luis-soares-ritense — APPROVED** [review]

File: release-notes/major9/09.24.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 The following features were added:
 
-* **New feature1**
+* **Search fields configuration deployment from resources**
 
-  Description of the new feature goes here.
-  Also link to the page that explains the feature in greater detail.
+  All search fields configurations that are placed in the resource folder, are now automatically deployed. More
```
Comment (marijnritense) [inline release-notes/major9/09.24.0/valtimo-backend-libraries.md:9]: Can you change the first sentence to:

All search fields configurations placed in the resource folder are automatically deployed.

File: using-valtimo/document/case-search.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      "path": "/customer/lastName",
+      "dataType": "text",
+      "fieldType": "single",
+      "matchType": "like"
+    }
+  ]
+}
+```
+
+**key**
```
Comment (marijnritense) [inline using-valtimo/document/case-search.md:34]: Bit unsure on this formatting. Is this consistent with how we do it for other pages?

File: using-valtimo/document/case-search.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Case search
```
Comment (marijnritense) [inline using-valtimo/document/case-search.md:1]: Missing an introduction

File: using-valtimo/document/case-search.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+```
+
+**key**
+
+This is an identifying human-readable key of the field.
+
+**path**
+
+The path pointing to the property that we want to search for. This field is optional.
```
Comment (marijnritense) [inline using-valtimo/document/case-search.md:40]: We?

File: using-valtimo/document/case-search.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+There are three field types: single, multiple, range. When a search field is configured to have the 'single' field type,
+a user can only search for one value at once. With the 'multiple' field type, a user can search for multiple
+values and with the 'range' field type, a user can search for a range of values.
+
+**matchType**
+
+There are two match types: exact and like. With the 'exact' match type, a user can only find cases that match exactly
+the search query. For example, if a user searches for cases with a last name property value 'Doe', it will not return
+cases that have a last name 'Doely'. But, when the match type 'like' is used, both cases with 'Doe' and 'Doely' are
```
Comment (marijnritense) [inline using-valtimo/document/case-search.md:57]: `But, ` can be removed.

File: using-valtimo/document/case-search.md (hunk 2 above)
Comment (Klaas-Ritense) [reply using-valtimo/document/case-search.md:37]: We don't do this for other pages as far as I know

**Pull request merged**